EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -1018,6 +1038,8 @@
 				OTHER_CFLAGS = (
 					"-fobjc-arc",
 					"-fobjc-arc-exceptions",
+					"-DNS_BLOCK_ASSERTIONS=1",
```

Comment:
> is -DXXX necessary?

I checked and you're correct, the build works without it.

> @bootstraponline, is there a way by which we could add these parameters in any other way, using a custom script or the scheme or so.

There might be, but not that I know of.